src/status_im/chat/events/commands.cljs

```diff
@@ -68,3 +75,12 @@
      {:dispatch [:request-permissions {:permissions [:read-external-storage]
                                        :on-allowed  #(re-frame/dispatch [:initialize-geth])}]}
      (log/debug "ignoring command: " command-name))))
+
+;; NOTE(goranjovic) - continues execution of a command that was paused by a shortcut
+(handlers/register-handler-fx
+ :execute-stored-command
+ [re-frame/trim-v]
+ (fn [{:keys [db]} []]
+   (let [{:keys [message opts]} (:commands/stored-command db)]
+     (-> (request-command-message-data db message opts)
+         (dissoc db :commands/stored-command)))))
```

Comment:
should it be just `(dissoc :commands/stored-command)`?

Reply:
I must be blind